torch/csrc/jit/script/init.cpp

```diff
@@ -385,6 +385,15 @@ void initJitScriptBindings(PyObject* module) {
         return std::make_tuple(py::bytes(module), python_serialized_export_map);
       }, py::arg("onnx_opset_version")=0,
          py::arg("operator_export_type")=::torch::onnx::OperatorExportTypes::RAW)
+      .def("export_to_pytorch_file", [](const std::shared_ptr<Module> m, const std::string& filename,
+                                        int64_t onnx_opset_version,
```

Comment:
why does it have onnx opset version? will it actually work without onnx export path?